.gitlab-ci.yml

```diff
@@ -131,24 +131,29 @@ release-prod-tag:
   <<: *docker-hub-login
   stage: release-public
   tags: ["runner:docker"]
-  image: registry.ddbuild.io/docker-notary:0.6.1
   script:
-    - docker pull registry.ddbuild.io/${CONTROLLER_IMAGE_NAME}:${TAG}
-    - docker tag registry.ddbuild.io/${CONTROLLER_IMAGE_NAME}:${TAG} datadog/${CONTROLLER_IMAGE_NAME}:${TAG}
+    - docker buildx create --use
+    - *install-make
+    - >
+      make docker-build-only-all \
```

Comment:
This does have some repeated config from building the internal images, but given the number of differences, I felt this was more readable than trying to add three more env vars for controlling the config.